app/services/key_rotator.rb

```diff
@@ -0,0 +1,8 @@
+module KeyRotator
+  extend ActiveSupport::Concern
+
+  def old_keys(config_name)
+    queue_name = "#{config_name}_queue"
+    JSON.parse(Figaro.env.send(queue_name.to_sym) || '[]')
```

Comment:
And maybe it would be better for grepping if we didnt' interpolate at all, ex just pass in `'email_encryption_key'` as the param? (`send` works with strings too)